wordpress-mobile/WordPress-Android
# [Voice to Content] Initial UI

Part of:
https://github.com/Automattic/wordpress-mobile/issues/81 
https://github.com/Automattic/wordpress-mobile/issues/83

This PR include the Base UI work for the Voice To Content Flow
- Identifies `VoiceToContentUiState` and associated types
- Flushes out strings
- Compose screens and elements
- Content styles
- Basic waveform (needs a bunch of work)
- Bottom sheet height set to 60% of the screen


> [!IMPORTANT]
> This PR does not include 
> - A polished UI
> - A working waveform
> - Elapsed time viewer
> - Launch the Edit Post view
> - Proper error handling
> - Network connectivity checks
> - Unit tests for the VM and UseCase - these will follow post error handling

-----
## Testing instructions:
There is no official testing for this PR; however you can test the success flow by
- Installing, launching, and logging into the app using your a8c account or an account that has AI requests available for a site
- Select a site in which you have available AI requests
- Navigate to Me > Debug Settings
- Enable voice_to_content and restart the app
- Navigate to My Site > Tap the FAB
- Select "Post from Audio"
- Tap the mic icon to begin recording
- Say something :)
- Tap the stop icon
- ✅ Verify the processing view is shown
- The bottom sheet should close and look for "***=> result is " in the logs

-----

## Regression Notes

1. Potential unintended areas of impact
The voice to content flow is not working as expected

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

-----

## PR Submission Checklist: N/A

-----

## Testing Checklist (strike-out the not-applying and unnecessary ones): N/A
